tests/src/python/test_selective_masking.py

```diff
@@ -1358,7 +1358,9 @@ def test_markerline_masked(self):
         circle_symbol = QgsMarkerSymbol.createSimple({'size': '3'})
         sl.setSubSymbol(circle_symbol)
 
-        self.lines_layer.setRenderer(QgsSingleSymbolRenderer(QgsLineSymbol([sl])))
```

Comment:
@troopa81 FYI, this was apparently fixed in newer sip versions, so eventually this approach will work 😝

Reply:
So that explain why it works on my computer (Debian SIP 6.8.3) and not in CI (SIP 4.19.25)